google/cloud/firestore/field_path.h

```diff
@@ -183,6 +139,26 @@ class FieldPath {
    */
   bool valid_;
 };
+
+bool operator==(FieldPath const& lhs, FieldPath const& rhs);
+bool operator<(FieldPath const& lhs, FieldPath const& rhs);
```

Comment:
It looks like you forgot to move the comment blocks as well.

Reply:
I dropped them intentionally (although I'm willing to listen to reasons for keeping them).  My rationale is that they fall under the "simple and obvious" clause of the style guide for function comments, since the meaning of the operators is well-known and the comments were just reinforcing that. Actually, the comments make me wonder "is this doing something unexpected?", which is forbidden by the style guide: "Define overloaded operators only if their meaning is obvious, unsurprising, and consistent with the corresponding built-in operators".